Onlinversion des Programmpapiers

sebastianriese opened the pull request:
Fügt eine Onlineversion des Programmpapiers und die Printversion des Programmpapiers (unter `media/fsfw-programm.pdf`) zur Website hinzu.
@majewsky

sebastianriese committed:
Erster Entwurf: Eigene Seite für das Programm

Diskussionsbedarf:
* PDF link ist wahrscheinlich noch zu subtil (mit Beschriftung nach
  rechts rücken, oder klein und in Klammern unter die Überschrift)
* Ist der Abstract so sinnvoll – eventuell mit Trennlinien arbeiten.

sebastianriese committed:
add improved build of the programme paper

* uses microtype (fixes overfull hboxes and terrible puncutation
  jitter)
* the "Unterstützer" section was removed (and will be added again
  as soon as there are some)

sebastianriese committed:
Comment out the "Unterstützer" section

* comment out the "Unterstützer" section in the online version of
  the programme paper

sebastianriese committed:
Gedankenstrich repariert

* Geviertstrich zu Halbgeviertstrich

sebastianriese committed:
Programm: Besserer Printversionslink

* eventuell ist die Accessibility so nicht perfekt

majewsky committed:
Merge branch 'master' into programme-whitepaper

majewsky commented:
@sebastianriese Was hältst du davon, wenn wir das Header-Bild von der themen.html (dort mit `<h1>Unsere Themen</h1>` beschriftet) auch auf diese Seite übernehmen, damit es nicht so farblos wirkt?

Das Einzige, wo ich mir da noch nicht so ganz klar wäre, ist die Platzierung des PDF-Links, sodass der gegen das farbige Bild nicht untergeht.

majewsky commented on programm.html:
```diff
@@ -0,0 +1,99 @@
+            <h1>Unser Programm
+                <aside class="header-link">
+                    <a href="./media/fsfw-programm.pdf">
+                        Printversion <img src="./img/pdf.png" style="vertical-align: middle" alt="PDF logo">
```
`s/Printversion/Druckfassung/`, IMHO

Printversion geht zwar als deutsches Wort durch, aber ich hab mich instinktiv gefragt, warum das da auf Englisch steht, wenn der Rest der Seite deutsch ist.

sebastianriese commented:
Ja, gerne auch Druckfassung.

Gegen das Hintergrundbild ist nichts einzuwenden, dann muss man wahrscheinlich dem Druckfassungslink einen weißen opaken Hintergrund geben (aber ich könnte mir vorstellen, dass das zu unauffällig wird). Das heißt ich fürchte, das wird noch ein größerer Designaufwand.

majewsky committed:
s/Printversion/Druckfassung/

majewsky committed:
accessibility: add a subtle :active/:focus/:hover effect to the sitelogo

majewsky committed:
new mobile navigation setup

To cope with the increased amount of text links in the navigation bar,
we now break the icon links out into a second floating container on
narrow (i.e. phone) screens.

majewsky committed:
integrate imageheader and imagecredit footer into programm.html

majewsky committed:
adjust wording for single image credit

majewsky committed:
gah

majewsky commented:
Ready to :ship: from my POV

sebastianriese commented:
Thanks, then I'll merge this now.
